<ui version="4.0" >
 <class>FiberWidgetCtrlClass</class>
 <widget class="QWidget" name="FiberWidgetCtrlClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FiberWidgetCtrl</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnOperation" >
     <property name="text" >
      <string>Operation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnSelMode" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="text" >
      <string>Selection Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnClrMode" >
     <property name="text" >
      <string>Color Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnDispMode" >
     <property name="text" >
      <string>Display Mode</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
